app/src/main/res/values/strings.xml

```diff
@@ -2,14 +2,11 @@
     <string name="app_name">SkiBuddy</string>
     <string name="action_search">Search</string>
     <string name="action_settings">Settings</string>
-<<<<<<< HEAD
```

Comment:
aargh i just deleted this .. let me see how not to do this


On Wed, Dec 2, 2015 at 12:52 AM, Ernst Haagsman <notifications@github.com>
wrote:

> In app/src/main/res/values/strings.xml
> <https://github.com/darshlakhani/SkiBuddy/pull/10#discussion_r46389200>:
>
> > @@ -2,14 +2,11 @@
> >      <string name="app_name">SkiBuddy</string>
> >      <string name="action_search">Search</string>
> >      <string name="action_settings">Settings</string>
> > -<<<<<<< HEAD
>
> Dude, what is this? :P
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/darshlakhani/SkiBuddy/pull/10/files#r46389200>.
>



-- 
Thank You
Darshit Lakhani
Email: darshlakhani@gmail.com
Phone: (669)221-9748